### PR TITLE
Data-inview - Improved documentation and demo

### LINKED ISSUE #7754: data-inview - CSS Transition - "show-none" work in reverse
The [data-inview docs](http://wet-boew.github.io/wet-boew/docs/ref/data-inview/data-inview-en.html) said, that when ```show-none``` class is added, the transition should occur when the element is fully displayed. 

Currently this is the behavior I got in my testing (v4.0.22):
* with class ```show-none``` : Transition occur when the element is partially displayed
* without class  ```show-none```: Transition only occur when the element is fully displayed

So I do not got the behavior as described in the docs.

Here a sample of code for reproducing the issue :

```xml
<p>Ut porttitor, sem eget porta mattis, massa ligula malesuada sem, ac mattis massa mi ut mi. Pellentesque malesuada nec nisi sed interdum. Nunc blandit laoreet elementum. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vivamus nec nibh sit amet leo efficitur tristique. Pellentesque in auctor magna. Donec iaculis et orci vel venenatis. Donec pellentesque pharetra felis, non sagittis metus semper vel. Mauris vel tellus ipsum. Integer vestibulum est sem, sit amet maximus sapien vehicula et.</p>

<blockquote id="fademe" class="fade">

	<p><code>fade</code>: Effect show start when the next paragraph is partially displayed</p>
	<p>[...]posuere purus erat, ut sagittis est rhoncus a. Pellentesque dictum odio lectus, vel sodales [...]</p>

</blockquote>

<p class="wb-inview" data-inview="fademe">Vestibulum venenatis lorem ac est dapibus, dictum sagittis neque vehicula. Etiam pharetra est in turpis aliquam sollicitudin. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam scelerisque ullamcorper lacus et blandit. Vivamus ac tortor eget nisl gravida bibendum vitae sit amet ante. Nullam id mi nec nunc feugiat tincidunt sit amet quis risus. Curabitur faucibus, diam quis aliquet tempus, neque nulla iaculis justo, ut porta tellus leo faucibus magna. Nullam eros odio, dapibus sit amet nisl non, viverra ultricies diam. Sed tristique et elit dapibus facilisis.</p>

<p>Vivamus arcu ipsum, blandit a leo vitae, sagittis posuere arcu. Nulla in leo nec massa varius venenatis. Morbi maximus porta arcu, sed accumsan lectus rutrum sed. Proin est lectus, ultricies et porta tempus, facilisis sit amet libero. Donec ut lacus consequat, dignissim libero eget, dictum ante. Pellentesque venenatis et lacus a ullamcorper. Nunc porta dictum pretium. Proin id ornare tortor.</p>

<blockquote id="fademe2" class="fade">

	<p><code>fade</code>: Effect show start when the next section is fully displayed</p>
	<p>[...]posuere purus erat, ut sagittis est rhoncus a. Pellentesque dictum odio lectus, vel sodales [...]</p>

</blockquote>

<p class="wb-inview show-none" data-inview="fademe2">Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Nulla non est vitae odio tristique tincidunt. Ut aliquam eleifend sapien, ut porta sapien faucibus eu. Nunc aliquet ultricies ex at lobortis. Ut vel velit ac nisi consectetur tempus. Suspendisse tempus ante pretium neque molestie, vel bibendum ex varius. Quisque id arcu scelerisque, sodales velit sit amet, vulputate ante. Etiam quis neque vitae enim scelerisque porta. In ullamcorper nisl sem, a rutrum odio blandit a. Mauris condimentum lobortis posuere. Pellentesque commodo quam nec euismod dictum. Nullam fermentum lacus nec turpis venenatis malesuada. Vivamus id nunc vehicula, mollis lectus vitae, molestie risus. Maecenas eget ullamcorper nisl. Proin efficitur metus leo, nec porttitor erat tincidunt sed. In elit nunc, consequat et diam eget, tincidunt scelerisque mauris.</p>

<p>Nulla facilisi. Proin at ornare turpis. Maecenas ornare sagittis velit. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin feugiat leo sed velit fringilla gravida. Integer erat magna, vehicula a pulvinar at, elementum sit amet orci. Nam vitae semper enim. Donec dictum imperdiet purus vel luctus. Integer pretium tincidunt augue, et ullamcorper urna luctus eu. Curabitur ut tempor turpis. Ut mattis vel ante quis volutpat. Mauris at rutrum neque. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur non ligula tellus.</p>
```

### PR DESCRIPTION
**Pull request by duboisp**
close #7754

### CODE REVIEW

File: src/plugins/data-inview/data-inview-en.hbs [hunk 1]
```diff
@@ -42,11 +39,11 @@
 		</details>
 	</section>
 
-	<p>Different example text. Different example text. Different example text. Different example text. Different example text. Different example text. Different example text. Different example text.</p>
+	<p role="presentation">Different example text. Different example text. Different example text. Different example text. Different example text. Different example text. Different example text. Different example text.</p>
```
Comment (LaurentGoderre) [inline src/plugins/data-inview/data-inview-en.hbs]: why the `role=presentation`? Wouldn't that remove the semantic of the div?

Comment (duboisp) [inline src/plugins/data-inview/data-inview-en.hbs]: I did some testing with NVDA and I will change them for ```role="img" aria-label=""```. I will update my PR tonight.

Comment (LaurentGoderre) [inline src/plugins/data-inview/data-inview-en.hbs]: Why are you trying to suppress it though? Because it's filler test?

Comment (duboisp) [inline src/plugins/data-inview/data-inview-en.hbs]: It's clear that filler text is purely decorative and do not provide any useful information. In fact, it is just occupying some space to provide an user experience on how the plugin work.

Comment (LaurentGoderre) [inline src/plugins/data-inview/data-inview-en.hbs]: Alright, hope nobody starts using this for actual pages though

**Pull request merged**